src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/Message.java

```diff
@@ -32,6 +35,13 @@
         .takeWhile(segment -> !segment.startsWith(MessageTrailer.KEY))
         .collect(toList()));
 
+    @Getter(lazy = true)
+    private final Optional<PatientClinicalInfo> patientClinicalInfo = PatientClinicalInfo.createOptional(
```

Comment:
I'm not sure if this belongs here. Is Segment 10 not part of Segment 6 (InvestigationSubject)?

Reply:
Actually just noticed this already exists in `InvestigationSubject.java`

Reply:
Good point. Fixed in 9f7c67cf97f358baf81d0dd2028cacc49ac0eba3